pkg/auth/cookie.go

```diff
@@ -17,13 +17,18 @@ import (
 
 const (
 	// #nosec
-	accessTokenCookieName = "flyte_jwt"
+	accessTokenCookieName = "flyte_at"
```

Comment:
is this a backwards incompatible change?

Reply:
it'll force a re-auth... which is fine... correctness FTW